### DISCUSSION TITLE
[Feature Request] Set custom search providers by URL/etc (for homepage search)

### DISCUSSION THREAD

**Discussion by Joshndroid** [linuxserver/Heimdall#671] [Ideas]
Firstly, great start page/organization program.

What I am requesting is the ability to search in the home page with a custom search provider, something that I can potentially host myself, like searx, etc.

This would provide the benefit of being browser agnostic as long as I can point my browser to the home page.
This also will help in cases such as mobile browser not allowing particular search engines from being used.

**Comment by shulist** [comment]
Would it also be possible to allow for adding an `IP:Port` option for those of us who self host our search? I use Whoogle as a search provider.

**Comment by KodeStar** [comment]
Done in 2.3.0

**Reply by B0rax** [reply]
can you tell us how it works? the wiki page is not very helpful (at least to me).

Which file do I have to modify? Where do I enter the Adress?

**Reply by muttleydosomething** [reply]
I'm using version 2.4.13 and it certainly doesn't look like it from here. Could you be kind enough to spend a few more milliseconds and actually explain how to enter the IP address and port number because to me it looks like it's just a static dropdown list.

Thanks
